CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.28.0 / 2019-01-24
+
+* [FEATURE] CLI tool to lint YAML against CRD defintions (#2269)
```

Comment:
Not sure this should be part of the Prometheus Operator changelog, as it seems to be a tool on the side?

Reply:
Yep, it's a separate tool, however as it is directly related to the CRDs and to validating those, therefore I feel like it's a valid addition to the changelog. No hard feelings though.

Reply:
I am fine with either way.